models/DataObject/Classificationstore.php

```diff
@@ -111,6 +111,10 @@ public function setItems($items)
      */
     public function getItems()
     {
+        $doGetInheritedValues = Model\DataObject::doGetInheritedValues();
```

Comment:
using DataObject:doGetInheritedValues(); to avoid breaking Changes

 has it is done in getLocalizedKeyValue()